server/db.js

```diff
@@ -32,8 +31,8 @@ export const disconnectDb = () => pool.end();
  * `await db.query("<SQL>", [...<variables>])`.
  */
 export default {
-	query: (...args) => {
-		logger.debug("Postgres querying %O", args);
-		return pool.query.apply(pool, args);
+	query: (text, params) => {
+		logger.debug("Postgres querying %O", { text, params });
+		return pool.query(text, params);
```

Comment:
I don't think this is the same thing. Why did you change it? Are you sure this doesn't break anything?

Reply:
I have changed it back to the way it was